atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
Do you want me to change the current code or is the worker->store_metadata_in_osv3->orchestrator->koji_import work flow sufficient for now?

I think if we want to insert orchestrate_build into the process, it should be done the same way for both media_version and help_file and there should be a single small story to unify that handling.